src/components/buttonPanel.js

```diff
@@ -0,0 +1,37 @@
+import Button from './button';
+
+const ButtonPanel = () => (
+  <div className="button-container">
+    <div className="top-buttons">
+      <Button name="AC" property="basic-op" />
+      <Button name="+/-" property="basic-op" />
+      <Button name="%" property="basic-op" />
+      <Button name="/" property="basic-op" />
+    </div>
+    <div className="second-buttons">
+      <Button name="7" property="nums" />
+      <Button name="8" property="nums" />
+      <Button name="9" property="nums" />
+      <Button name="*" property="basic-op" />
```

Comment:
## Change `times` button sign on the screen display
- [ ] Change the `times` sign on the calculator display from `*` to `x` to adhere to the guidelines set forth. See the suggestion below 👇 
```suggestion
      <Button name="x" property="basic-op" />
```